configuration/pih/petl/jobs/general/diagnosis/source.sql

```diff
@@ -0,0 +1,52 @@
+select encounter_type_id into @tb_general from encounter_type where uuid= 'aa42cc6c-b9ee-4850-926c-dda4bb14d890';/*50 tb intake*/
+select encounter_type_id into @covid_general from encounter_type where uuid='8d50b938-dcf9-4b8e-9938-e625bd2f0a81';/*45 covid-19 admission*/
+select encounter_type_id into @hiv_general from encounter_type where uuid='c31d306a-40c4-11e7-a919-92ebcb67fe33';/*29 hiv intake*/
+
+drop temporary table if exists temp_diagnosis;
+create temporary table temp_diagnosis
+(
+especialidad  varchar(20),
+frecuencia_dias int,
+frecuencia_semana int,
+frecuencia_mes int,
+frecuencia_manana int,
+frecuencia_tarde int
+);
+
+set @YEAR=YEAR(NOW());
+
+insert into temp_diagnosis (especialidad,frecuencia_dias,frecuencia_semana,frecuencia_mes,frecuencia_manana,frecuencia_tarde)
+select 'tuberculosis',(count(encounter_type))/251 as frecuencia_dia,
```

Comment:
Is this because there are 251 business days in a year?

Reply:
Exactly.